### ISSUE TITLE
Is it Possible to Generate MP4s from a set of Images?

### ISSUE THREAD

**Issue by Camuvingian** [hudl/HudlFfmpeg#71]
I would just like to know if this project can generate MP4s (for example) from a set of frames/image files? I have looked through the example/tests and could not see anything that suggest this was possible...

**Comment by Casey-Bateman** [comment]
@Camuvingian  yes it can, what you would do is add the the input as a glob patter. so if you have pngs your input uri would look like this `@"c:\foo\bar%03d.png"`

**Comment by qcjxberin** [comment]
could you give a sample
